### ISSUE TITLE
Crafting Plan : Crafting CPU : Automatic , Start buttons greyed out

### ISSUE THREAD

**Issue by Voidfulqt** [AppliedEnergistics/Applied-Energistics-2#618]
Hello , 
     I'm running the lastest version of the Crackpack modpack from ATLauncher ( The Crack Pack 2.0.1.0-TCP ) Single Player and I recently encountered a strange problem. When I login into the game everything works fine but after a while everything I try to craft simply "crashes" if I can say that , the buttons in the Crafting Plan ( Crafting CPU : Automatic and Start ) are simply greyed out and can't craft anything. I have around 20 crafting cpus the largest one is ~2M storage and 68 Co-processors.
I'm not sure when did this happen but recently I expanded my ME Controllers and I've made a cuboid shape max size 7x7x7 that is.
       My networks are probably okay using around 8 dense cables and every cable is in the 32 max channels range. The only dense cable which is 32 of 32 is my molecular assembler one. I tried making a separate network with less than 8 channels just a simple one with a molecular assembler and an interface and then it worked. I tried connecting the dense cables each of them to a different controller from the multi-block structure ( instead of having 2-3 cables connecting to different sides of a single controller block ) but that didn't work. I'm not sure what to say more , maybe I did something wrong , but I'm pretty sure I've did everything in order. 
      Any help is much appreciated , pretty much now I'm stuck with crafting and I can't do anything. Best regards.

**Comment by desagas** [comment]
I have the exact same issue, on nearly everything. At first I could only craft once in 5 to 10 clicks, but now it says automatic all of the time and start is greyed out exactly as the image shown above. I had it in a server, but i read somewhere that servers can cause this when sending data to a client, so I moved my world to single player, and the exact same issue occurs. I play on Modsauce 1.10 and have quite a big setup, and replicating it would be nearly impossible, and since you do not want a actual world it happens in, it is very hard to show you. As modsauce is running stable 1 AE2, perhaps this has already been fixed. I hope so. I can not wait until they update to the newest release.

To thatslch, if the crafting cpu was tied up doing other things, it would say No-CPU's available would it not?

I have now checked everything, from all my p2p tunnels being resetup to ensure they are all correct, to making sure that there are clear separation between cable to no cause any conflict of channels.

I can not wait until the next stable release of AE2
